packages/create-cloudflare/src/cli.ts

```diff
@@ -158,7 +158,7 @@ const configure = async (ctx: C3Context) => {
 
 	// Note: updateWranglerToml _must_ be called before the configure phase since
```

Comment:
This comment needs updating, it can probably be updated not to include the function name as it does now to avoid the possibility to get it out of sync with it